src/sentry/static/sentry/app/components/letar.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import MD5 from 'crypto-js/md5';
+
+/**
+ * Also see avatar.py. Anything changed in this file (how colors
+ * are selected, the svg, etc) will also need to be changed there.
+ */
+
+const COLORS = {
+  blue: '#25A6F7',
+  blue_dark: '#1D87CE',
+  green: '#6FBA57',
+  green_dark: '#4F923C',
+  yellow_orange: '#F8A509',
+  red: '#E35141',
+  red_dark: '#B64236',
+  pink: '#E56AA6',
+  purple: '#836CC2',
+  purple_dark: '#6958A2',
+  teal: '#44ADA0',
+  gray: '#6F7E94'
+};
+
+const Letar = React.createClass({
+  propTypes: {
+    user: React.PropTypes.object.isRequired
+  },
+
+  COLORS: COLORS,
+
+  COLOR_COUNT: Object.keys(COLORS).length,
+
+  getColor() {
+    let userId = Math.floor((this.getIdentifierHash() / 4096) * this.COLOR_COUNT);
+    let color = 'blue';
+    if (userId % 12 == 0) {
```

Comment:
btw. Looking over this again, this is actually not doing what you want. At least I'm pretty sure it does not.

I think you are looking for this:

```js
const colors = Object.values(COLORS);
let color = colors[userId % this.COLOR_COUNT];
```